### PR TITLE
*: add experimental warnings

### PR DESCRIPTION
**Pull request by jackysp**
Signed-off-by: Shuaipeng Yu <jackysp@gmail.com>

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3330, https://github.com/pingcap/docs-cn/pull/3664
- Other reference link(s):

### CODE REVIEW

File: production-deployment-using-tiup.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ aliases: ['/docs/dev/how-to/deploy/orchestrated/tiup/','/docs/dev/tiflash/deploy
 
 # Deploy a TiDB Cluster Using TiUP
 
+> **Warning:**
+>
+> This is still an experimental feature. It is **NOT** recommended that you use this attribute in the production environment.
+
```
Comment (yikeke) [inline production-deployment-using-tiup.md:13]: This is added in the wrong location. @jackysp

Comment (jackysp) [reply production-deployment-using-tiup.md:13]: ah, yes...

Comment (jackysp) [reply production-deployment-using-tiup.md]: PTAL @yikeke

File: configure-placement-rules.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ aliases: ['/docs/dev/how-to/configure/placement-rules/']
 
 # Placement Rules
 
+> **Warning:**
```
Comment (yikeke) [inline configure-placement-rules.md:10]: See https://github.com/pingcap/docs-cn/pull/3330/files#r440069140

File: dashboard/dashboard-intro.md [hunk 3]
```diff
@@ -6,6 +6,10 @@ category: how-to
 
 # TiDB Dashboard Introduction
 
+> **Warning:**
+>
+> This is still an experimental feature. It is **NOT** recommended that you use this attribute in the production environment.
```
Comment (yikeke) [inline dashboard/dashboard-intro.md:11]:
```suggestion
> TiDB Dashboard is still an experimental feature. It is **NOT** recommended that you use it in the production environment.
```

File: sql-statements/sql-statement-create-index.md [hunk 4]
```diff
@@ -84,6 +84,10 @@ Query OK, 0 rows affected (0.31 sec)
 
 ## Expression index
 
+> **Warning:**
+>
+> This is still an experimental feature. It is **NOT** recommended that you use this attribute in the production environment.
```
Comment (yikeke) [inline sql-statements/sql-statement-create-index.md:89]:
```suggestion
> Expression index is still an experimental feature. It is **NOT** recommended that you use it in the production environment.
```

File: system-tables/system-table-sql-diagnosis.md [hunk 5]
```diff
@@ -7,6 +7,10 @@ aliases: ['/docs/dev/reference/system-databases/sql-diagnosis/']
 
 # SQL Diagnosis
 
+> **Warning:**
+>
+> This is still an experimental feature. It is **NOT** recommended that you use this attribute in the production environment.
```
Comment (yikeke) [inline system-tables/system-table-sql-diagnosis.md:12]:
```suggestion
> SQL diagnosis is still an experimental feature. It is **NOT** recommended that you use it in the production environment.
```

File: tidb-configuration-file.md [hunk 6]
```diff
@@ -46,6 +46,10 @@ The TiDB configuration file supports more options than command-line parameters.
 
 ### `oom-action`
 
+> **Warning:**
+>
+> This is still an experimental feature. It is **NOT** recommended that you use this attribute in the production environment.
```
Comment (yikeke) [inline tidb-configuration-file.md:51]: This is not aligned with [docs-cn changes](https://docs.pingcap.com/zh/tidb/v4.0/tidb-configuration-file#oom-action). @jackysp

File: tidb-configuration-file.md [hunk 7]
```diff
@@ -46,6 +46,10 @@ The TiDB configuration file supports more options than command-line parameters.
...
+
 - Specifies what operation TiDB performs when a single SQL statement exceeds the memory quota specified by `mem-quota-query` and cannot be spilled over to disk.
 - Default value: `"cancel"`
```
Comment (yikeke) [inline tidb-configuration-file.md:54]:
```suggestion
- Default value: `"log"`
```

File: tikv-configuration-file.md [hunk 8]
```diff
@@ -95,6 +95,10 @@ This document only describes the parameters that are not included in command-lin
 
 ## readpool.unified
 
+> **Warning:**
+>
+> This is still an experimental feature. It is **NOT** recommended that you use this attribute in the production environment.
```
Comment (yikeke) [inline tikv-configuration-file.md:100]:
```suggestion
> Unified read pool is still an experimental feature. It is **NOT** recommended that you use it in the production environment.
```

File: configure-placement-rules.md (hunk 2 above)
Comment (jackysp) [reply configure-placement-rules.md:10]: done

File: tidb-configuration-file.md (hunk 6 above)
Comment (jackysp) [reply tidb-configuration-file.md:51]: done

**Comment by jackysp** [comment]
PTAL @yikeke

File: configure-placement-rules.md [hunk 9]
```diff
@@ -7,6 +7,10 @@ aliases: ['/docs/dev/how-to/configure/placement-rules/']
...
+>
+> In the scenario of using TiFlash, the Placement Rules has been extensively tested and can be used in a production environment. Except for the scenarios where TiFlash is used, using Placement Rules alone has not been extensively tested, so it is not recommended to enable this feature separately in a production environment.
```
Comment (yikeke) [inline configure-placement-rules.md:12]:
```suggestion
> In the scenario of using TiFlash, the Placement Rules feature has been extensively tested and can be used in the production environment. Except for the scenario where TiFlash is used, using Placement Rules alone has not been extensively tested, so it is not recommended to enable this feature separately in the production environment.
```

File: dashboard/dashboard-intro.md [hunk 10]
```diff
@@ -6,6 +6,10 @@ category: how-to
 
 # TiDB Dashboard Introduction
 
+> **Warning:**
```
Comment (yikeke) [inline dashboard/dashboard-intro.md:9]: This is not from https://github.com/pingcap/docs-cn/pull/3330/files. Where does this warning come from? @jackysp

File: tidb-configuration-file.md [hunk 11]
```diff
@@ -46,8 +46,12 @@ The TiDB configuration file supports more options than command-line parameters.
 
 ### `oom-action`
 
+> **Warning:**
+>
+> This function currently counts the memory during the writing process, which is an experimental feature. For users who want to rely on this feature to cancel the write operation, it is not recommended to configure it as cancel in the production environment.
```
Comment (yikeke) [inline tidb-configuration-file.md:51]:
```suggestion
> This feature is still an experimental feature, which counts the memory during the writing process. For users who want to use this feature to cancel the write operation, it is not recommended to configure it as `cancel` in the production environment.
```

File: dashboard/dashboard-intro.md (hunk 10 above)
Comment (jackysp) [reply dashboard/dashboard-intro.md:9]: https://github.com/pingcap/docs-cn/pull/3330/files#diff-c9747b847859e83c802693a25cb125c2R1351

Comment (yikeke) [reply dashboard/dashboard-intro.md:9]: The note should be in post-installation-check.md. See https://github.com/pingcap/docs/pull/2832/commits/16ce338c64886bef9c94fa2801d0e5d453db0727

Comment (yikeke) [reply dashboard/dashboard-intro.md:9]: Decided to keep this note in dashboard/dashboard-intro.md. I will add this note in docs-cn later.

Comment (yikeke) [reply dashboard/dashboard-intro.md:9]: Updated this note in https://github.com/pingcap/docs-cn/pull/3675

**Pull request merged**